### PR TITLE
Fix: Display margin below min deposit amount

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
1. Show the balance even if the available margin is below the minimum deposit amount (50 sUSD), e.g.
```
Only $18.54 available margin 
Min. $50 sUSD required to trade
```
2. Simplify the existing codes

## Related issue
Continue the work on #2631

## How Has This Been Tested?
Case 1: The user has 0 sUSD in the available margin
- See `No available margin`
- Click on `Get sUSD` button to open `Deposit/Withdraw` modal

Case 2: The user has less than 50 sUSD in the available margin
- See `Only $xx.xx available margin`
- Click on the pencil button to open `Deposit/Withdraw` modal

Case 3: The user has more than 50 sUSD in the available margin
- See `Available margin: $xxx.xx`
- Click on the pencil button to open `Deposit/Withdraw` modal

## Screenshots (if appropriate):
***Desktop***
截屏2023-07-25 15 21 24

***Mobile***
截屏2023-07-25 15 20 53

### CODE REVIEW

File: packages/app/src/sections/futures/Trade/TradeBalance.tsx [hunk 1]
```diff
@@ -29,19 +30,50 @@ import CrossMarginInfoBox from '../TradeCrossMargin/CrossMarginInfoBox'
...
+		</FlexDivRowCentered>
+	)
+}
+
+const TradeBalance = memo(() => {
 	const { t } = useTranslation()
 	const dispatch = useAppDispatch()
+
 	const { deviceType } = useWindowSize()
+	const isMobile = deviceType === 'mobile'
```
Comment (avclarke) [inline packages/app/src/sections/futures/Trade/TradeBalance.tsx:68]: Could we wrap these in memo and move below selectors

**Review by avclarke — COMMENTED** [review]
Looks good, just a small bit of clean up

File: packages/app/src/sections/futures/Trade/TradeBalance.tsx (hunk 1 above)
Comment (LeifuChen) [reply packages/app/src/sections/futures/Trade/TradeBalance.tsx]: Update. Thanks.

File: packages/app/src/sections/futures/Trade/TradeBalance.tsx [hunk 2]
```diff
@@ -29,19 +30,50 @@ import CrossMarginInfoBox from '../TradeCrossMargin/CrossMarginInfoBox'
...
+	const size = isMobile ? 'small' : 'medium'
+
+	const accountType = useAppSelector(selectFuturesType)
+	const isCrossMarginAccount = accountType === 'cross_margin'
```
[the remark this answers is not in the document]
Comment (LeifuChen) [reply packages/app/src/sections/futures/Trade/TradeBalance.tsx]: Update. Thanks.

**Review by platschi — APPROVED** [review]

**Pull request merged**